.github/CODEOWNERS

```diff
@@ -1,8 +1,8 @@
 # Team Lead
-* @rosle
+* @malparty
 
 # Team Members
-* @andyduong1920 @bterone @byhbt @carryall @hanam1ni @hoangmirs @junan @Lahphim @longnd @malparty @tyrro
+* @rosle @andyduong1920 @bterone @byhbt @carryall @hanam1ni @hoangmirs @junan @Lahphim @longnd @tyrro
```

Comment:
These lines should be:
```suggestion
# Team
# @malparty is the Team Lead and the others are team members
* @malparty @andyduong1920 @bterone @byhbt @carryall @hanam1ni @hoangmirs @junan @Lahphim @longnd @rosle @tyrro
```

As these 2 lines will apply rules to `*` (any files), the latter will override the previous one. We discussed it once in [gin-templates](https://github.com/nimblehq/gin-templates/pull/75#pullrequestreview-930100069) 😄

Reply:
:harold: 
I totally forgot about this >< Thanks!

Reply:
846c2b6 ;-)